### ISSUE TITLE
A-softmax loss with 64 layer?

### ISSUE THREAD

**Issue by double-vane** [wy1iu/sphereface#13]
First of all thank you for your open code. You have developed a 20-layer Sphereface framework, and Is it necessary to study the 64-layer? Which of the two is better?

**Comment by wy1iu** [comment]
The 64-layer network consistently performs better than the 20-layer network. According to the paper, it should be easy to modify the SphereFace-20 to SphereFace-64. We will also release the 64-layer network soon. Thanks.

**Comment by double-vane** [comment]
Thank you very much for receiving your reply!
I first trained the 64-layer model with softmax loss, but I failed in finetuning A-softmax .The model does not converge.
Look forward to your model release.

Best!

**Comment by wy1iu** [comment]
You could start a new issue if you have further questions.

**Comment by anguoyang** [comment]
Hi@double-vane have you trained the 64-layer successfully,could you please share the model? thank you

**Comment by smajida** [comment]
Hi @double-vane could you please share the model? thank you
